lib/common/delete-phases-common.js

```diff
@@ -59,9 +60,16 @@ exports.unDeployCloudFormationStack = function (serviceContext, serviceType) {
             }
         })
         .then(() => {
+          return s3Calls.deleteMatchingPrefix('handel-'+accountConfig.region+'-'+accountConfig.account_id,serviceContext.appName+'/'+serviceContext.environmentName+'/'+serviceContext.serviceName);
+          return;
```

Comment:
This extra return should be removed, it will be treated as an error by linters since it's unreachable code.